pkg/cli/server/server.go

```diff
@@ -132,7 +132,18 @@ func run(app *cli.Context, cfg *cmds.Server) error {
 		serverConfig.ControlConfig.DefaultLocalStoragePath = cfg.DefaultLocalStoragePath
 	}
 
+	noDeploys := make([]string, 0)
 	for _, noDeploy := range app.StringSlice("no-deploy") {
+		if strings.Contains(noDeploy, ",") {
```

Comment:
Thanks for the PR!

```suggestion
```
Might be better to remove the if/else logic and just append in the loop, if `noDeploy` does not contain `,` then `strings.Split` will create a slice of length 1.

Reply:
Right. Makes sense. I'll update.